### DISCUSSION TITLE
Add room selection to Roomba integration

### DISCUSSION THREAD

**Discussion by JQDigital** [home-assistant/frontend#14911] [Configuration] [answered]
The current Roomba integration is very basic and contains only the following commands:
- Start
- Stop
- Return to dock

However, the robovac has evolved dramatically and this basic integration is faling behind.
The integration should be extended to contain at least roomselection, zoneselection

Additionally the whole map feature would be great, so it is able to create and edit: zones, rooms, and avoidspaces

**Comment by frenck** [comment, marked as answer]
Hi there @JQWeb 👋 

This is the GitHub discussion for the Home Assistant frontend; however, your discussion is about integration. Please use the Home Assistant community forums for feature requests like these:

<https://community.home-assistant.io>

../Frenck
